kairon/api/app/routers/bot/data.py

```
@@ -370,6 +371,42 @@
+    media_id = await MongoProcessor.upload_media_to_bsp(
+        bot = current_user.get_bot(),
+        user = current_user.get_user(),
+        file_path = file_path,
+        file_info = file_content,
+    )
+
+    return Response(message = "File uploaded successfully!", data = media_id)
+
+@router.get("/upload/media_upload", response_model=Response)
+async def get_media_ids(
+    current_user: User = Security(Authentication.get_current_user_and_bot, scopes=DESIGNER_ACCESS),
```

Ruff (0.12.2)

Do not perform function call `Security` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)